tests/cpp_api_tests/rocAL_video_unittests/rocAL_video_unittests.cpp

```diff
@@ -189,7 +210,7 @@ int main(int argc, const char **argv)
         default:
         {
             std::cout << "\n>>>> VIDEO READER\n";
-            input1 = rocalVideoFileSource(handle, source_path, color_format, decoder_mode, shard_count, sequence_length, shuffle, is_output, false, frame_step, frame_stride, file_list_frame_num);
+            input1 = rocalVideoFileSource(handle, source_path, color_format, decoder_mode, shard_count, sequence_length, {}, shuffle, is_output, false, frame_step, frame_stride, file_list_frame_num);
```

Comment:
instead of passing an empty structure, please pass 0 or nullptr which can be used if passed as reference